_data/students.yml

```diff
@@ -1,3 +1,9 @@
+ -name: Chirag Kumar
```

Comment:
There must be a space after `-` that's why Travis is failing. Fix it and we are ready for a merge